src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,62 @@
+body {
+  background-color: #696969;
+}
+
+.watch {
+  width: $watch-size;
+  height: $watch-size;
+  border-radius: 50%;
+
+  background: #286160;
+  border: 4px solid #090118;
+
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: $watch-transform;
+
+  &::before {
+    @include watch-hands-position;
+
+    content: "";
+    width: $big-hand-width;
+    height: $big-hand-height;
+    left: ($watch-size / 2) - ($big-hand-width / 2);
+    top: ($watch-size / 100 * 19);
```

Comment:
What are these numbers? Hard to guess, better create variables for them (or calculate from other vars)